docs/quarantine.md

```diff
@@ -102,3 +102,30 @@ After merging this PR the test will be out of quarantine.
 [2]: https://www.thoughtworks.com/en-us/insights/blog/no-more-flaky-tests-go-team
 [3]: https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html#quarantined-tests
 [on testgrid]: https://testgrid.k8s.io/kubevirt-periodics
+
+# Test Lane Quarantine
+
+There can be cases where required test lanes are flaking in a clustered manner with a large
+number of tests failing each time. This can cause major delays to merging important pull
+requests and overload CI. Individual test case quarantining does not make sense in this case.
+
+A required test lane should be made optional if the following criteria are met:
+* The required test lane has a failure rate higher than 25% in the last 24 hours
```

Comment:
I'd suggest to add a link to where to look that percentage up.

Reply:
+1

We can even make use of the new https://github.com/kubevirt/ci-health?tab=readme-ov-file#failures-per-sig-against-last-code-push-for-merged-prs

Reply:
> I'd suggest to add a link to where to look that percentage up.

I suggested using the CI search to calculate the impact of the flake over the last 24 hours 
https://github.com/kubevirt/kubevirt/pull/12494/files#diff-4ff5f85030a93d12b9e7929720bd3d43b5f73cba313b3e98a56dbefe2d2abfabR118

Reply:
> +1
> 
> We can even make use of the new https://github.com/kubevirt/ci-health?tab=readme-ov-file#failures-per-sig-against-last-code-push-for-merged-prs

These counts are for the last 7 days and only apply to merged PRs so it doesn't show flakes that open PRs in the queue may be hitting.

Reply:
> I am not sure if they are the best fit for this.

Unlike counting failures of open PRs, your kubevirt-ci metric is not polluted by bad code. An open PR may be failing for a very good reason - the new PR is bad.

If a lane is so bad that it does not let you merge PRs at all, it would be easy to notice. I think that our biggest problem these days is when a PR (modifying either test or production code or both) ends up increasing failure rate by 10%. I human developer would likely not notice that before a PR is merged, even by searching for failures.
However, your new metric would show a clear bump after a week.

Yes, it is slow, but it is undeniable, and can help us call out the relevant SIG to fix the issue. I worry that the semi-manual search for errors require too much explanation and negotiation.

Maybe the policy can mention both: *either* you find to many failures in error logs *or* the lane metrics jumps above 25%? Either way, the lane should be marked non-voting until fixed by the relevant SIG.

Reply:
Yes fair points - I've added a link to the top failed jobs list - https://github.com/kubevirt/kubevirt/pull/12494/commits/b083974607c80ac6000d54b947eb33001ec594c0